src/main/java/com/aws/greengrass/mqtt/bridge/clients/MQTTClient.java

```diff
@@ -203,12 +203,25 @@ public void stop() {
             LOGGER.atDebug().setCause(e).log("Unable to close mqtt client datastore");
         }
 
+        IMqttClient client = null;
         try {
-            IMqttClient client = mqttClientInternal;
```

Comment:
try the disconnectForcibly option instead of doing a nice disconnect in our `disconnect` method.

Reply:
yep that worked well, skips a bunch of checks that we preventing disconnect and sent the disconnect packet right away